RUFAS/biophysical/manure/storage/solids_storage_calculator.py

```diff
@@ -209,3 +227,68 @@ def calculate_anaerobic_coefficient() -> float:
             (OXYGEN_HALF_SATURATION_CONSTANT + AMBIENT_AIR_MOLE_FRACTION_OF_OXYGEN)
             / AMBIENT_AIR_MOLE_FRACTION_OF_OXYGEN
         )
+
+    @staticmethod
+    def calculate_ifsm_methane_emission(manure_volatile_solids: float, manure_temp: float) -> float:
```

Comment:
```suggestion
    def calculate_ifsm_methane_emission(manure_volatile_solids: float, manure_temperature: float) -> float:
```
I think we're trying not to abbreviate words where possible because things like `temp` could mean `temperature` or `temporary` for example.